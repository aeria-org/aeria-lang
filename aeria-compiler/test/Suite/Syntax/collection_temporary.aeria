collection CollectionTemporaryTest {
  temporary {
    index prop1
    expireAfterSeconds 2
  }
}
