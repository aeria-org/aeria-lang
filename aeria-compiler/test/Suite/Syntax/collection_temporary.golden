{
  "kind": "Program",
  "collections": [
    {
      "kind": "Collection",
      "name": {
        "value": "collectiontemporarytest",
        "kind": "CollectionName"
      },
      "temporary": {
        "kind": "CollectionTemporary",
        "index": {
          "value": "prop1",
          "kind": "PropertyName"
        },
        "expireAfterSeconds": 2
      }
    }
  ]
}
